cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,68 @@
 /// <reference types='cypress' />
+import { user } from "../support/testData";
+import { book } from "../support/testData";
 
 describe('Book Store app', () => {
   before(() => {
-    
+    cy.visit('/login')
+    cy.viewport(1488, 1000)
+  });
+
+  beforeEach(() => {
+    cy.viewport(1200, 1000)
+  });
+
+
+  it.skip('user should be able to login', () => {
```

Comment:
Consider removing the `.skip` from the login test to ensure it is being executed.